src/main/java/duke/ui/Parser.java

```diff
@@ -0,0 +1,9 @@
+package duke.ui;
+
+import duke.commands.Command;
+
+public class Parser {
+    public static Command parseCommand(String input) {
+        return new Command(input);
+    }
+}
```

Comment:
Maybe you could place the functions related to parsing user string input here instead of putting it inside the Command class.